src/ontology/pato-edit.obo

```diff
@@ -22478,8 +22484,3 @@ name: singly_occurring_form_of
 comment: PATO divides qualities between normal (monadic, singly-occurring) qualities and relational qualities. Relational qualities stand in the 'towards' relation with respect to some additional entity. For example, The sensitivity of an eye towards red light. In some cases we want to represent a quality such as 'protruding' in both monadic and relational branches. We use this relation to link them.
 synonym: "monadic_form_of" EXACT []
 
-[Typedef]
-id: towards
```

Comment:
Shouldnt this be obsoleted properly?

Reply:
Did not obsolete, but instead replaced it by adding a RO ID on it - happy to properly obsolete this too if we should do it that way